src/webview/js/generatedEs3.js

```diff
@@ -537,13 +537,13 @@ var isNearPositions = function isNearPositions() {
 
 documentBody.addEventListener('touchend', function (e) {
   if (isNearPositions(lastTouchPositionX, lastTouchPositionY, e.changedTouches[0].pageX, e.changedTouches[0].pageY)) {
-    lastTouchEventTimestamp = Date.now();
+    touchActive = false;
```

Comment:
The `Date.now()` here wasn't required, as the `!lastTouchEventTimestamp` would capture the cases it covered anyways if it was set to `0` instead.